src/Common/ORDataObject/ORDataObject.php

```diff
@@ -5,24 +5,26 @@
  *
  */
 
+namespace OpenEMR\Common\ORDataObject;
+
 class ORDataObject
 {
-    var $_prefix;
-    var $_table;
-    var $_db;
+    private $prefix;
+    private $table;
+    private $db;
 
-    function __construct()
+    public function __construct()
     {
-        $this->_db = $GLOBALS['adodb']['db'];
+        $this->db = $GLOBALS['adodb']['db'];
```

Comment:
hi @yashrajbothra ,
why ended up removing all the underline characters?
Definitely some of those changes will break things. Would rec keeping them.

Reply:
There is a warning when I keep those underscore.
![image](https://user-images.githubusercontent.com/47683556/77884379-8f142b00-7282-11ea-91c2-d4db60b085ca.png)

I was afraid of Travis so I removed those. WIll revert and push again 👍

Reply:
Done! If Travis is happy then we can take this in 😃